contracts/staking/contracts/src/staking_pools/MixinCumulativeRewards.sol

```diff
@@ -264,59 +260,108 @@ contract MixinCumulativeRewards is
     )
         internal
         view
-        returns (uint256)
+        returns (uint256 reward)
     {
-        // sanity check inputs
         if (memberStakeOverInterval == 0) {
             return 0;
         }
 
-        // sanity check interval
-        if (beginEpoch >= endEpoch) {
+        // Sanity check interval
+        if (beginEpoch > endEpoch) {
             LibRichErrors.rrevert(
                 LibStakingRichErrors.CumulativeRewardIntervalError(
-                    LibStakingRichErrors.CumulativeRewardIntervalErrorCode.BeginEpochMustBeLessThanEndEpoch,
+                    LibStakingRichErrors
+                        .CumulativeRewardIntervalErrorCode
+                        .BeginEpochMustBeLessThanEndEpoch,
                     poolId,
                     beginEpoch,
                     endEpoch
                 )
             );
         }
 
-        // sanity check begin reward
-        IStructs.Fraction memory beginReward = _cumulativeRewardsByPool[poolId][beginEpoch];
+        // Sanity check begin reward
+        IStructs.Fraction memory beginReward =
+            _cumulativeRewardsByPool[poolId][beginEpoch];
         if (!_isCumulativeRewardSet(beginReward)) {
             LibRichErrors.rrevert(
                 LibStakingRichErrors.CumulativeRewardIntervalError(
-                    LibStakingRichErrors.CumulativeRewardIntervalErrorCode.BeginEpochDoesNotHaveReward,
+                    LibStakingRichErrors
+                        .CumulativeRewardIntervalErrorCode
+                        .BeginEpochDoesNotHaveReward,
                     poolId,
                     beginEpoch,
                     endEpoch
                 )
             );
         }
 
-        // sanity check end reward
-        IStructs.Fraction memory endReward = _cumulativeRewardsByPool[poolId][endEpoch];
+        // Sanity check end reward
+        IStructs.Fraction memory endReward =
+            _cumulativeRewardsByPool[poolId][endEpoch];
         if (!_isCumulativeRewardSet(endReward)) {
             LibRichErrors.rrevert(
                 LibStakingRichErrors.CumulativeRewardIntervalError(
-                    LibStakingRichErrors.CumulativeRewardIntervalErrorCode.EndEpochDoesNotHaveReward,
+                    LibStakingRichErrors
+                        .CumulativeRewardIntervalErrorCode
+                        .EndEpochDoesNotHaveReward,
                     poolId,
                     beginEpoch,
                     endEpoch
                 )
             );
         }
 
-        // compute reward
-        uint256 reward = LibFractions.scaleFractionalDifference(
+        // Compute reward
+        reward = LibFractions.scaleFractionalDifference(
             endReward.numerator,
             endReward.denominator,
             beginReward.numerator,
             beginReward.denominator,
             memberStakeOverInterval
         );
-        return reward;
```

Comment:
I like explicitly returning personally. We have been trying to do it throughout the codebase.

Reply:
Yeah, I liked the conciseness originally but I agree it's better overall to be explicit.